internal/person/repository.go

```diff
@@ -32,9 +31,6 @@ func (repo *personRepository) GetProfile(ctx context.Context, iamID string) (*mo
 		models.PersonWhere.IamID.EQ(iamID)).One(ctx, repo.executor)
 
 	if err != nil {
-		if err == sql.ErrNoRows {
```

Comment:
I think this is the correct place to throw error, as its `sql.ErrNoRows` and `repo` is handling DB records :)